exec/java-exec/src/main/java/org/apache/drill/exec/planner/common/DrillRelOptUtil.java

```diff
@@ -603,7 +606,7 @@ public static boolean guessRows(RelNode rel) {
             || (tableMetadata = table.getGroupScan().getTableMetadata()) == null
             || !TableStatisticsKind.HAS_DESCRIPTIVE_STATISTICS.getValue(tableMetadata);
       } catch (IOException e) {
-        RelOptPlanner.LOGGER.debug("Unable to obtain table metadata due to exception:", e);
+        logger.debug("Unable to obtain table metadata due to exception: {}", e.getMessage(), e);
```

Comment:
@vvysotskyi is this change ok? Or `RelOptPlanner` was used intensionally?

Reply:
Yes, using a specific logger is more suitable. Thanks for fixing this!